src/dht/dht_router.cc

```diff
@@ -76,16 +76,11 @@ DhtRouter::DhtRouter(const Object& cache, const rak::socket_address* sa) :
   }
 
   if (m_nodes.size() < num_bootstrap_complete) {
-    m_contacts = std::make_unique<std::deque<contact_t>>();
-
+    m_contacts.emplace();
     if (cache.has_key("contacts")) {
-      const Object::list_type& contacts = cache.get_key_list("contacts");
-
-      for (const auto& contact : contacts) {
-        auto               litr = contact.as_list().begin();
-        const std::string& host = litr->as_string();
-        int port = (++litr)->as_value();
-        m_contacts->emplace_back(host, port);
+      for (const auto& contact : cache.get_key_list("contacts")) {
+        auto litr = contact.as_list().begin();
+        m_contacts->emplace_back(litr->as_string(), std::next(litr)->as_value());
```

Comment:
The host/port variables in the original code serve as comments, and are therefore preferred imo. The code output is the same.

Reply:
OK